diesel/src/query_dsl/boxed_dsl.rs

```diff
@@ -57,14 +57,14 @@ impl<'a, T, DB> InternalBoxedDsl<'a, DB> for T where
 /// # }
 /// ```
 ///
-/// Diesel queries also have a similar problem to [Iterator][iterator], where
+/// Diesel queries also have a similar problem to [`Iterator`][iterator], where
 /// returning them from a function requires exposing the implementation of that
-/// function. The [helper_types][helper_types] module exists to help with this,
+/// function. The [`helper_types`][helpertypes] module exists to help with this,
 /// but you might want to hide the return type or have it conditionally change.
 /// Boxing can achieve both.
 ///
 /// [iterator]: https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html
-/// [helper_types]: ../helper_types/index.html
+/// [helpertypes]: ../helper_types/index.html
```

Comment:
Oh, I tried making sense of the doc_markdown lint which wanted me to put
helper_types in back ticks. I didn't mean to leave that in. (Is there a
clippy issue for that? I should probably create one otherwise)
Sean Griffin <notifications@github.com> schrieb am Mo., 25. Apr. 2016 um
23:24:

> In diesel/src/query_dsl/boxed_dsl.rs
> <https://github.com/diesel-rs/diesel/pull/305#discussion_r60992194>:
>
> >  /// but you might want to hide the return type or have it conditionally change.
> >  /// Boxing can achieve both.
> >  ///
> >  /// [iterator]: https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html
> > -/// [helper_types]: ../helper_types/index.html
> > +/// [helpertypes]: ../helper_types/index.html
>
> Lol what
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/diesel-rs/diesel/pull/305/files/9b4d3b77b7c5c046956b10be79757433abbec0b2#r60992194>
>